modules/about-rhde.adoc

```diff
@@ -81,6 +87,9 @@ For more information about the {product-title} products, see the following docum
 
 * link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/{ansible-version}[Product Documentation for {ansible}]
 
+//* link:https://access.redhat.com/documentation/en-us/edge_management/2023[Product Documentation for Edge management 2023]
+//the RHEL team owns the edge management page; date change was to apply before GA, but no other pages than 2022 exist as of 25Oct2023
+
```

Comment:
```suggestion
```
That page looks super funky and unmaintained now, see https://docs.redhat.com/en/documentation/edge_management/1-latest

We can just delete even the commented-out references to it.